schedule/yast/autoyast_salt_formulas.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: autoyast_salt_formulas
+description: |
+  Test installation using AutoYaST plus salt formulas.
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/validate_salt_formula
```

Comment:
validation should be at the end, so we have those test in the middle that we add depending of our needs and the validations are the last thing to do, but it is kind of optional, it is a visually observation.